.vscode/tasks.json

```diff
@@ -23,7 +23,6 @@
         "/consoleloggerparameters:NoSummary"
       ],
       "problemMatcher": "$msCompile",
-      "dependsOn": ["checkout:sample:locks"]
```

Comment:
We could probably drop the "checkout:sample:locks" as well as it assumes one will have cloned the samples repo. 
But I'd like to get the view of @baywet on this as too.

Reply:
yep, I wanted to make my life simpler when debugging the generated clients. But I don't do it nearly as often as I used to, so the description hash is usually enough now.

Reply:
Does this mean we remove the task "checkout:sample:locks" altogether?

Reply:
yes, I did a quick search and there doesn't seem to be a way to disable a task when a directory does not exist or to ignore when it fails.
We could try that before removing it all together https://stackoverflow.com/a/72567713/3808675

Reply:
I have tried the || for both the command level and args level but getting errors. So resulted to just removing the task as earlier agreed.

Reply:
thanks for looking into it